challenges/objects-arrays.js

```diff
@@ -7,11 +7,32 @@
 */
 
 // tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceious
-
+const tyrannosaurus = {
+  name: 'tyrannosaurus',
+  diet: 'carnivorous',
+  weight: '7000kg',
+  length: '12m',
+  period: 'Late Cretaceious',
+  roar : function(){
```

Comment:
Declare roar outside of the tyrannosaurus object declaration.

Reply:
Check into this article on method definitions to make your method syntax shorter: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions

:)